ext/dom/tests/DOMXPath_clone.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+DOMXPath: Cloning a DOMXPath object
+--EXTENSIONS--
+dom
+--SKIPIF--
+<?php
+if (!class_exists('DOMXPath')) die('skip no xpath support');
```

Comment:
When DOM ext is present, `DOMXPath` can be unavailable?

Reply:
It's possible to compile libxml2 without xpath support. In that case, DOMXPath will be unavailable as well.
However, I'm not aware of any distro that does this, and it is not the default configuration of libxml2.

Reply:
Would it make sense to make DOMXPath required using configure?

Reply:
I'd lean towards yes, but you could ask the mailing list if anyone actually is on a system that purposefully disabled xpath support in libxml2.